src/components/schemas/expense.yaml

```diff
@@ -373,7 +373,8 @@ expense_out:
     employee_id:
       $ref: './fields.yaml#/employee_id'
     employee:
-      $ref: './employee.yaml#/employee_out_embed'
+      nullable: true
+      $ref: './employee.yaml#/employee_out_embed_for_expense'
```

Comment:
This is intended.
- Updating employee_out_embed will force the re-write of data & test cases across the whole repo
- API contracts will be changed for out of scope APIs
- For reports also its done like this recently
- Will create a separate task for above
- https://app.clickup.com/t/85ztjmjdh